sugarcrm/client.py

```diff
@@ -16,6 +16,9 @@ def _api_request(self, method, args):
         if not self.session_id and method != "login":
             self._login()
 
+        if self.session_id:
```

Comment:
wouldn't this be the `else` of the `if` above?

Reply:
actually not, bcs the self.session_id is set during the self._login()  so for example when called with the search_email method, it needs to enter the first and the second if.

it could be replace instead by 
`if method != "login":
`

Reply:
ah, I see.